specification/adp/resource-manager/Microsoft.AutonomousDevelopmentPlatform/preview/2020-07-01-preview/adp.json

```diff
@@ -674,6 +674,7 @@
             "Succeeded",
             "Failed",
             "Canceled",
+            "Accepted",
```

Comment:
Why is this enum marked as `"modelAsString": false`?

Reply:
I followed the same pattern for `ProvisioningState` from other RPs. Why do you think it should it be `true`? I would like an enum to be generated in the SDK.

Reply:
Closed enums (modelAsString: false) have proven to be version intolerant (over time). If you can commit to never, ever add a new provisioning state value, then this is ok.